specs/darwin/password_policy.table

```diff
@@ -0,0 +1,11 @@
+table_name("password_policy")
```

Comment:
At least on my mac, this is documented both as having global policies, as well as per-user policies. Any thoughts about how to handle that?

Reply:
Yes, the reason I used `ODNodeCopyAccountPolicies` API call is to get both global and per-user policies. I am not sure on other platform, but windows has `NetUserModalsGet` in it's win32 API, from their description:


>The NetUserModalsGet function retrieves global information for all users and global groups in the security database, which is the security accounts manager (SAM) database or, in the case of domain controllers, the Active Directory.

From: https://docs.microsoft.com/en-us/windows/win32/api/lmaccess/nf-lmaccess-netusermodalsget?redirectedfrom=MSDN

I am guessing that should cover it? I am not sure on Linux

Reply:
On macOS, the concept of `user` policies and `global` policies has been deprecated for a while now. From the man page:

>  Account Policies
     Account policies are the replacement for the deprecated legacy global and user policies.  Account policies are specified as a dictionary containing three keys, one key for each policy category.  Note that the dictionary is
     not required to contain all of the policy categories
     
The `policy_identifier` is something that a user can set (either manually or via MDM). Apple supplied policy has identifiers that start with `com.apple.`, and currently Apple only supplies one policy by default -- where if filevault gets enabled, the password has to be 4 characters or more.

For other platforms, if there is a clear user/global distinction, I guess one can use the extended schema for the table.